tests/10apidoc/11profile-avatar_url.pl

```diff
@@ -1,6 +1,6 @@
 my $user_fixture = local_user_fixture();
 
-my $avatar_url = "http://somewhere/my-pic.jpg";
+my $avatar_url = "mxc://example.com/SEsfnsuifSDFSSEF";
```

Comment:
Heads up this definitely isn't the only occurrence of an `http://` URL being used as an avatar in the codebase.